docs/THEOplayer/results/issues.md

```diff
@@ -0,0 +1,41 @@
+# Issues
+
+## Issue #1
+
+Live Stream with Low Latency
+
+Feature is expected to be supported at the end of 2020.
+
+------
+
+## Issue #2
+
+Ubuntu
+
+THEOplayer does not support this platform.
+
+------
+
+## Issue #3
+
+HLS streams on webOS
+
+HLS streams won't start or keeps buffering after a few seconds. This has been reported internally and will be fixed in due course.
+
+------
+
+## Issue #4
+
+DRM on tvOS
+
+DRM on tvOS does not work at all. This has been reported internally and will be fixed in due course.
+
+------
+
+## Issue #5
+
+HLS streams on Tizen
+
+HLS streams won't start or keeps buffering after a few seconds. This has been reported internally and will be fixed in due course.
```

Comment:
Is there a tracking issue link?

Reply:
Unfortunately, we do not have our tracking link exposed publicly (only internally). However, once the issue has been addressed, we can update our observations here.